source/exercises/mailroom.rst

```diff
@@ -243,29 +250,28 @@ When you are done, push your changes and make a pull request.
 .. _exercise_mailroom_plus:
 
 
-Part 2: Adding dicts and Files
+Part 2: Adding dicts and Files (Lesson 4)
 ==============================
```

Comment:
when you make a heading with underlining, the underline is supposed to be the full length of the text above it.

I'll go fix that now.